src/js/main.js

```diff
@@ -395,37 +400,84 @@ Site.Map = {
     var newX = _this.mapPosition[4] + (translation * Math.cos(angle));
     var newY = _this.mapPosition[5] + (translation * Math.sin(angle));
 
+    // Move the map
+    _this.moveMap(newX,newY);
+
+    // Animate recursevly
+    window.requestAnimationFrame(_this.pan.bind(_this));
+
+  },
+
+  // Check if golocation is available and move the map to that equivalent location
+  geolocation: function() {
+    var _this =  this;
+
+    if (WP.clientGeolocation === '0') {
```

Comment:
If we `return false` from the php we can check for `WP.clientGeolocation == false` here